data/card_catalogue/unique-1.json

```diff
@@ -112,7 +112,7 @@
         "cast": "Target willing character ceases to exist for up to 1 hour. They return to the last spot they existed in afterwards.",
         "hand": null
       },
-      "flavour": "Existence is rather tiring, it's advisable to take a break every once in a while",
+      "flavour": "Existence is tiring, why not take a break?",
```

Comment:
Thoughts on just `Existence is tiring`?

Reply:
It feels a little obtuse to me by itself